include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -309,9 +309,8 @@ class format_help : public format_base
         for (unsigned i = 0; i < meta.synopsis.size(); ++i)
         {
             std::string text = "\\fB";
-            text.append(meta.app_name);
-            text.append("\\fP ");
             text.append(meta.synopsis[i]);
+            text.replace(text.find_first_of(" \t"), 0, "\\fP");
```

Comment:
why `\t` ?

Reply:
I oriented on `is_blank` function that considers tab and space.
When used properly it's probably not necessary here. However, it's also no significant extra cost afaik. So I decided to include it.

Reply:
Hm I don't know if we need this but I guess it is fine.
What's the second parameter to replace doing?

Reply:
> What's the second parameter to replace doing?

The length of replacement, e.g.:

```
std::string str "hello world";
str.replace(str.find_first_of(" "), 0, ""); // hello world
str.replace(str.find_first_of(" "), 1, ""); // helloworld
str.replace(str.find_first_of(" "), 3, ""); // hellorld
```

Actually, now I see it, std::string::insert should also work and could be slightly faster (I guess), although theoretical complexity is the same:
```
std::string str "hello world";
str.insert(str.find_first_of(" ")+1, "new "); // hello new world
```

shall I change this?

Reply:
Well if one is concerned about performance we would not create temporary strings at all :grin: 
but we are not in the argument parser. `insert` is nicer though because the interface is better known so yes that would be nice